workflow_capability_core/src/main/java/com/philips/healthsuite/workflowcapability/core/demos/FhirStoreInitialization.java

```diff
@@ -26,16 +34,53 @@ public void run() {
         this.fhirUrl = this.env.getProperty("config.fhirUrl");
         this.fhirDataResources = new FhirDataResources(this.fhirUrl + "/fhir");
 
+        clearSubscriptions();
         addTaskSubscription();
         addCarePlanSubscription();
     }
 
+    public void clearSubscriptions() {
+        Subscription[] subscriptions = fhirDataResources.getSubscriptions();
+        for (Subscription subscription : subscriptions) {
+            List<Extension> extension = subscription.getExtension();
+
+            if(extension == null || extension.size() == 0) {
+                continue;
+            }
+
+            for(Extension ext : extension) {
+                boolean deleted = false;
+
+                if(ext.getUrl().equals("createdBy")) {

```

Comment:
Misuse of Extension in a Subscription Resource
Suggestion: Instead use a tag as part of the meta of a Resource